### DISCUSSION TITLE
when trying to run it returns this error

### DISCUSSION THREAD

**Discussion by IsoumaNaN** [captcha-challenger/hcaptcha-model-factory#50] [Q&A]
![image](https://user-images.githubusercontent.com/83762343/211841294-e97513a4-4ce2-407d-8b51-ea852e9100a2.png)

**Comment by QIN2DIM** [comment]
You can currently run the project using the `trainval` command, I think there are some potential bugs with the `new` command.
- https://github.com/captcha-challenger/hcaptcha-model-factory/issues/49
